OrbitCore/TracepointEventBuffer.h

```diff
@@ -34,9 +39,8 @@ class TracepointEventBuffer {
     }
   }
 
-  Mutex& mutex();
-
  private:
+  int32_t kAllTracepointsNotInTargetProcessFakeTid = -1;
```

Comment:
kNotTargetProcessThreadId?

Reply:
Of course! Thanks a lot for the suggestion!